WordPress/gutenberg
# Added Bluesky icon to the Social Icon Block

## What?
I have added the Bluesky icon to the Social Icons Block.

## Why?
Bluesky has just been added as a trusted oEmbed provider in core.

This is related to: #61325 

## How?


## Testing Instructions
1. Open a Post or Page
2. Add a Social Block
4. Add the Bluesky Icon
4. Add content to it, normally https://bsky.app/profile/your-username
5. Save and make sure it appears as expected in the editor as well as the preview

## Screenshots or screencast 

In the editor:
<img width="1244" alt="image" src="https://github.com/WordPress/gutenberg/assets/77401999/28a48b15-cfaf-410d-92f0-d73c677356aa">

Result in the preview:
<img width="639" alt="image" src="https://github.com/WordPress/gutenberg/assets/77401999/562ba213-0a11-4f2c-84bb-51c548b1e8af">
